src/clp_bench/clpg_executor.py

```
@@ -46,7 +46,7 @@ def ingest(self, mode: BenchmarkingMode):
             decompressed_size_mb = (
                 int(result.stdout.decode("utf-8").split("\n")[-2].split()[0].strip()) / 1024 / 1024
             )
-            self.benchmarking_reseults[mode].decompressed_size = f"{decompressed_size_mb:.2f}MB"
+            self.benchmarking_results[mode].decompressed_size = f"{decompressed_size_mb:.{BenchmarkingResult.SIZE_PRECISION}f}MB"
```

Ruff

Line too long (129 > 100)

(E501)